packages/controller/src/resources/app/api.ts

```diff
@@ -99,25 +102,25 @@ export function ConfigAPIResources(
           name,
           namespace,
           labels: {
-            "k8s-app": name
+            app: name
```

Comment:
I like this much better. We should lean into this convention and use it for all our deployments